gci/views.py

```diff
@@ -26,8 +26,9 @@ def index(request):
         student_url = STUDENT_URL.format(org_id=org_id,
                                          student_id=student_id,
                                          )
-        s.append('<li><a href="{student_url}">{student_id}</a>: '
-                 '<a href="https://github.com/{username}">{username}</a>'
+        s.append('<li><a href="{student_url}">{student_id}</a>:<br />'
```

Comment:
Also mention that these {student_id} are student ids, otherwise they'll look like just arbitrary numbers. 
Something like
```
<span>STUDENT ID: </span> {student_id} 
```